etc/config/python.defaults.properties

```diff
@@ -0,0 +1,18 @@
+compilers=&python3
+
+group.python3.compilers=python38:python37
+compiler.python38.name=Python 3.8
+compiler.python38.exe=/usr/local/bin/python3
```

Comment:
These paths are non-standard.

Maybe the best defaults would just be "python3.8" and "python3.7"?

I believe it should resolve them using the path variable, and the full version names are more appropriate than just "python3", which currently causes both of these to show up and both use the same version of python under the hood.